radio/src/myeeprom.h

```diff
@@ -491,16 +491,25 @@ enum UartModes {
 
 PACK(typedef struct {
   uint8_t type:4;
-  int8_t  rfProtocol:4;
+  int8_t  old_rfProtocol:4;
   uint8_t channelsStart;
-  int8_t  channelsCount; // 0=8 channels
-  uint8_t failsafeMode:7;
+  int8_t  channelsCount;  // 0=8 channels
+  uint8_t failsafeMode:4;
+  uint8_t subType:3;
   uint8_t invertedSerial:1; // telemetry serial inverted from standard
   int16_t failsafeChannels[NUM_CHNOUT];
-  int8_t  ppmDelay:6;
-  uint8_t ppmPulsePol:1;
-  uint8_t ppmOutputType:1;     // false = open drain, true = push pull
-  int8_t  ppmFrameLength;
+    union {
+        struct {
+            int8_t ppmDelay:6;
+            uint8_t ppmPulsePol:1;
+            uint8_t ppmOutputType:1;     // false = open drain, true = push pull
+            int8_t ppmFrameLength;
+        };
+        struct {
+            int8_t rfProtocol;
+            int8_t optionValue;
+        };
```

Comment:
I can't see how do you know which part of the union to use!?  For example when the `rfProtocol` is PPM.